packages/astro/test/i18n-routing.test.js

```diff
@@ -2087,4 +2087,12 @@ describe('Fallback rewrite SSR', () => {
 		const html = await response.text();
 		assert.match(html, /Hello/);
 	});
+
+	it('should render paths with path parameters correctly', async () => {
+		let request = new Request('http://example.com/new-site/fr/blog/1');
```

Comment:
Can we update the test with more locales? At the moment there's only `fr: 'en'`, let's add two more and make sure that the fallback of params works for all of them.